src/lib/version/version.c

```diff
@@ -206,13 +204,10 @@ const char *px4_firmware_git_branch(void)
 	return PX4_GIT_BRANCH_NAME;
 }
 
+#if !defined(__PX4_NUTTX)
```

Comment:
This whole block of code is implemented in a new file specifically for the Nuttx build. The API is unchanged.